Readme.md

```diff
@@ -1,3 +1,52 @@
-### Taskcluster AMQP Event Proxy
+### Taskcluster AMQP Events Proxy
+
+This proxy, hosted at `events.taskcluster.net/v1`, allows clients to receive events(pulse messages) on AMQP exchanges specified in [`taskcluster-client`](https://github.com/taskcluster/taskcluster-client). 
```

Comment:
`This proxy allows web-based applications to  consume Pulse messages, such as those produced by other Taskcluster services.`

(I want to not mention a specific hostname, for redeployability purposes)